superdesk/locators/__init__.py

```diff
@@ -24,5 +25,6 @@ def _load_json(file_path):
 
 
 _dir_name = os.path.dirname(os.path.realpath(__file__))
-_locators_file_path = os.path.join(_dir_name, 'data', 'locators.json')
+_locators_file = app.config.get('SUPERDESK_LOCATORS', 'locators.json')
```

Comment:
imo would be better to use the folder we have for init data, so you don't have to modify core when updating it